var/spack/repos/builtin/packages/py-audioread/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAudioread(PythonPackage):
+    """cross-library (GStreamer + Core Audio + MAD + FFmpeg) audio decoding for
+    Python."""
+
+    homepage = "https://github.com/beetbox/audioread"
+    url      = "https://pypi.io/packages/source/a/audioread/audioread-2.1.8.tar.gz"
+
+    version('2.1.8', sha256='073904fabc842881e07bd3e4a5776623535562f70b1655b635d22886168dd168')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
Is `py-pytest-runner` needed at build time? I see:
```python
      setup_requires=[                                                          
          'pytest-runner'                                                       
      ],  
```
in the `setup.py`

Reply:
I think it is for testing. I see no difference in the build with or without it. Should I add it?

Reply:
I went ahead and added that dependency just to be complete. The build logs do not show a difference.